From df9b55f54fd5927ff653a88787c860fb9b7f72d7 Mon Sep 17 00:00:00 2001
From: Jurgen Kramer <gtmkramer@xs4all.nl>
Date: Fri, 29 Apr 2016 12:47:57 +0200
Subject: [PATCH] usb: Add native DSD support for NuPrime HD-AVP/AVA

This patch adds native DSD support for the NuPrime Audio HD-AVP and HD-AVA.

Signed-off-by: Jurgen Kramer <gtmkramer@xs4all.nl>
---
 sound/usb/quirks.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/sound/usb/quirks.c b/sound/usb/quirks.c
index 0adfd95..bc818d4 100644
--- a/sound/usb/quirks.c
+++ b/sound/usb/quirks.c
@@ -1307,6 +1307,7 @@ u64 snd_usb_interface_dsd_format_quirks(struct snd_usb_audio *chip,
 			return SNDRV_PCM_FMTBIT_DSD_U32_BE;
 		break;
 
+	case USB_ID(0x16d0, 0x06b4): /* NuPrime Audio HD-AVP/AVA */
 	case USB_ID(0x20b1, 0x000a): /* Gustard DAC-X20U */
 	case USB_ID(0x20b1, 0x2009): /* DIYINHK DSD DXD 384kHz USB to I2S/DSD */
 	case USB_ID(0x20b1, 0x2023): /* JLsounds I2SoverUSB */
-- 
2.5.5

